Platform.PCL/Realm.PCL/Schema/RealmSchemaPCL.cs

```diff
@@ -38,7 +38,14 @@ public class RealmSchema : IReadOnlyCollection<ObjectSchema>
         /// Gets the number of known classes in the schema.
         /// </summary>
         /// <value>The number of known classes.</value>
-        public int Count { get; }
```

Comment:
For code to be able to get ahold of a `RealmSchema` instance to call `Count` on then the realm assembly must be loaded instead of the reference one.